test/org/jetbrains/plugins/ideavim/action/MotionActionTest.java

```diff
@@ -510,7 +510,7 @@ public void testDeleteInnerTagBlockBetween() {
   //|d| |v_it|
   public void testDeleteInnerTagBlockBetweenTagWithRegex() {
     typeTextInFile(parseKeys("dit"), "abcde<[abc]*>af<caret>gbc</[abc]*>hi");
-    myFixture.checkResult("abcde<[abc]*></[abc]*>hi");
+    myFixture.checkResult("abcde<[abc]*>afgbc</[abc]*>hi");
```

Comment:
I changed the test in accordance to the behavior I get in vanilla vim.
IMO the closest ideavim behave like vanilla vim the better.